CHANGELOG.md

```diff
@@ -1,17 +1,19 @@
 # Changelog
 
-## Unreleased
+## 0.2.0 (2020-01-28)
```

Comment:
Totally agree we should be using ISO for the dates.

Reply:
It's actually in https://keepachangelog.com/en/1.0.0/ I missed it for some reason 😄